ndn_hydra/repo/group_messages/heartbeat.py

```diff
@@ -13,14 +13,17 @@
 from ndn.encoding import *
 import json
 from ndn_hydra.repo.modules.global_view import GlobalView
+from ndn_hydra.repo.modules.favor_calculator import FavorCalculator
 from ndn_hydra.repo.group_messages.specific_message import SpecificMessage
 
 class HeartbeatMessageTypes:
     NODE_NAME = 84
-    FAVOR = 86
+    LATITUDE = 85
+    LONGITUDE = 86
 class HeartbeatMessageTlv(TlvModel):
     node_name = BytesField(HeartbeatMessageTypes.NODE_NAME)
-    favor = BytesField(HeartbeatMessageTypes.FAVOR)
+    latitude = BytesField(HeartbeatMessageTypes.LATITUDE)
+    longitude = BytesField(HeartbeatMessageTypes.LONGITUDE)
```

Comment:
First, we need a structure called FavorParamaters that includes lat/long and other information.
We can then include that struct in ALL messages via the ModelType tlv. In addition, we need to update the global view every message.

Reply:
Hi, please check out my most recent commit and let me know what you think. In regards to updating the global view, do you mean that just for the heartbeat messages? Or ALL messages? If you mean just the heartbeat, I believe in the apply function in the Heartbeat message it updates the global view with the new calculated favor.

![Screenshot of 4 nodes running with different coordinates in client](https://user-images.githubusercontent.com/62191506/191871872-df2410c0-8406-4da6-84e2-6ccb6f86ca4b.png)
![The node table in global view of node1](https://user-images.githubusercontent.com/62191506/191871947-51868ff0-f1ba-41f0-a333-85049d273447.png)